src/app/shared/object-collection/shared/badges/embargo-badge/embargo-badge.component.ts

```diff
@@ -0,0 +1,70 @@
+import {
+  AsyncPipe,
+  NgIf,
+} from '@angular/common';
+import {
+  Component,
+  Input,
+  OnInit,
+} from '@angular/core';
+import { TranslateModule } from '@ngx-translate/core';
+import {
+  catchError,
+  map,
+  Observable,
+  of as observableOf,
+} from 'rxjs';
+import { AccessStatusDataService } from 'src/app/core/data/access-status-data.service';
+import { Bitstream } from 'src/app/core/shared/bitstream.model';
+import { hasValue } from 'src/app/shared/empty.util';
+import { environment } from 'src/environments/environment';
+
+import { AccessStatusObject } from '../access-status-badge/access-status.model';
+
+@Component({
+  selector: 'ds-base-embargo-badge',
```

Comment:
My intuition is that we'll want to make this component more "generic" in the future (see my comments in the REST PR)

Could you adjust the name/selector already so we won't have to deal with that later?
e.g. `*-access-badge` instead of embargo specifically.

Reply:
OK, I did every changes requested on the angular side. Hope it's good enough.